templates/subscription.template.json

```diff
@@ -284,6 +305,7 @@
         }
       }
     ],
+    "loggingRedisCacheName": "[toLower(concat('das-', parameters('resourceEnvironmentName'),'-log-rds')]",
```

Comment:
Can this be moved up alongside the other resource name variables, you'll then also see that the others are using 'resourceNamePrefix' so you should do the same